selfdrive/car/toyota/interface.py

```diff
@@ -140,6 +152,8 @@ def get_params(candidate, fingerprint):
       ret.minEnableSpeed = -1.
     elif candidate in [CAR.RAV4, CAR.COROLLA]: # TODO: hack ICE to do stop and go
       ret.minEnableSpeed = 19. * CV.MPH_TO_MS
+    elif candidate == CAR.LEXUS_AVE30:
+      ret.minEnableSpeed = 46. * CV.KPH_TO_MS # MY17 is300h/AVE30R (AU ver.) does not have stop and go, minimum ACC speed allow is 46kph
```

Comment:
@EternityLan , I would keep this 19mph like for rav4 and corolla. Even though the stock acc won't engage below 28mph/46kph, the system still respond down to 18mph/30kph.